raiden/smart_contracts/ChannelManagerLibrary.sol

```diff
@@ -75,10 +99,11 @@ library ChannelManagerLibrary {
 
         self.node_channels[caller_address].push(channel_address);
         self.node_channels[partner].push(channel_address);
+        self.channel_addresses[partyHash(caller_address, partner)] = channel_address;
         self.all_channels.push(channel_address);
     }
 
-    /// @notice deleteChannel(address) to remove a channel after it's been settled
+    /// @notice deleteChannel(address, address, address, uint, uint) to remove a channel after it's been settled
```

Comment:
Additionally there is no need to provide the types in the notice tag.

Remember: Notice is what the end-user would be seeing when executing from mist. Dev is what the developer should read. You don't need to provide both of them if it's not a user-facing function.